src/main/java/com/google/sps/Utility.java

```diff
@@ -242,4 +318,31 @@ public static MultiMutation filterMultiMutationByNodes(MultiMutation mm, Set<Str
         .setReason(mm.getReason())
         .build();
   }
+
+  /**
+   * Given a list of node names, a map from node name to mutation indices of that node and a list of
+   * multimutations applied to all nodes, returns a list of indices of multimutations in which any
```

Comment:
"returns a set of indices". Also the sorted order comment doesn't make sense anymore since we're returning a set.